baselines/fedrep/fedrep/conf/base.yaml

```diff
@@ -0,0 +1,45 @@
+---
+num_clients: 100 # total number of clients
+num_local_epochs: 5 # number of local epochs
+num_rep_epochs: 1 # number of representation epochs (only for FedRep)
+enable_finetune: false
+# num_finetune_epochs: 10
+batch_size: 50
+num_rounds: 100
+learning_rate: 0.01
+momentum: 0.5
+algorithm: fedrep
+model_name: cnncifar10
+
+client_resources:
+  num_cpus: 4
+  num_gpus: 1
```

Comment:
I wonder if we could increase the degree of parallelism by lowering these. For example, even running 5x clients in parallel (done by lowering `num_gpus: 0.2` and then having a low enough `num_cpus` -- e.g set to 2 or 1 should do the trick) it requires just 2.6GB of VRAM

The same comment applies to the other configs

Reply:
Sounds interesting. I will try this out later today. Since the model used is small, this parallelism should be reasonable.